scenes/src/app/scenes/slider/slider-scene.scss

```diff
@@ -0,0 +1,10 @@
+mat-slider {
+  width: 300px;
+  margin-bottom: 15px;
+}
+
+.scene-sliders {
+  position: absolute;
```

Comment:
Rather than repeating the same question in every PR here I'll just ask this a second time and leave the others:
Is it possible to center these rather than trying to calculate a position with `absolute`?